src/ui-test/tests/deploy.kubernetes.run.test.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { resolve } from 'node:path';
+import { CAMEL_ROUTE_YAML_WITH_SPACE } from '../variables';
+import {
+    ActivityBar,
+    after,
+    before,
+    BottomBarPanel,
+    EditorAction,
+    EditorView,
+    repeat,
+    SideBarView,
+    VSBrowser,
+} from 'vscode-uitests-tooling';
+import { killTerminal, waitUntilTerminalHasText } from '../utils';
+import { execSync } from 'child_process';
+
+describe('Camel standalone file deployment using Camel JBang Kubernetes Run', function () {
+    this.timeout(800_000);
+
+    let editorView: EditorView;
+    const RESOURCES_PATH: string = resolve('src', 'ui-test', 'resources');
+
+    before(async function () {
+        await VSBrowser.instance.openResources(RESOURCES_PATH);
+        await (await new ActivityBar().getViewControl('Explorer')).openView();
+        const section = await new SideBarView().getContent().getSection('resources');
+        await section.openItem(CAMEL_ROUTE_YAML_WITH_SPACE);
+
+        editorView = new EditorView();
+        await repeat(async function () {
+            return (await editorView.getOpenEditorTitles()).find(title => title === CAMEL_ROUTE_YAML_WITH_SPACE);
+        }, {
+            timeout: 10_000,
+            message: `The test file ${CAMEL_ROUTE_YAML_WITH_SPACE} was not opened`
+        });
+    });
+
+    after(async function () {
+        await killTerminal();
+        await editorView.closeAllEditors();
+        // remove deployed integration from a local cluster
+        // TODO pay attention to change 4.9.0-SNAPSHOT to some stable version
```

Comment:
this can be changed by getting default Camel JBang version from settings, after there is a correct one to have kubernetes plugin working

Reply:
obsolete, 4.8.1 already as default version